### ISSUE TITLE
programmatically scroll?

### ISSUE THREAD

**Issue by taivu** [drusso85/ng2-scroll-to#18]
is there a way to programmatically call the scrolling methods? For example, after checking a checkbox, scroll somewhere else.

**Comment by sinsunsan** [comment]
Yes a service to  call the scroll from within another service or component would be a nice addition...
